p8Layouts/defaultPnorLayoutWithGoldenSide.xml

```diff
@@ -194,18 +194,18 @@ Layout Description
         <ecc/>
     </section>
     <section>
-        <description>Payload (1MB)</description>
+        <description>Payload (0.75MB)</description>
         <eyeCatch>PAYLOAD</eyeCatch>
         <physicalOffset>0xCA9000</physicalOffset>
-        <physicalRegionSize>0x100000</physicalRegionSize>
+        <physicalRegionSize>0xC0000</physicalRegionSize>
         <side>A</side>
         <readOnly/>
     </section>
     <section>
-        <description>Bootloader Kernel (15MB)</description>
+        <description>Bootloader Kernel (15.25MB)</description>
         <eyeCatch>BOOTKERNEL</eyeCatch>
-        <physicalOffset>0xDA9000</physicalOffset>
-        <physicalRegionSize>0xF00000</physicalRegionSize>
+        <physicalOffset>0xD69000</physicalOffset>
+        <physicalRegionSize>0xF40000</physicalRegionSize>
         <side>A</side>
         <readOnly/>
```

Comment:
layout with golden side has another pair of payload/bootkernel partitions to modify

Reply:
Fixed with new push.